crates/rpc/rpc-engine-api/src/engine_api.rs

```diff
@@ -736,13 +775,21 @@ where
         config: TransitionConfiguration,
     ) -> RpcResult<TransitionConfiguration> {
         trace!(target: "rpc::engine", "Serving engine_exchangeTransitionConfigurationV1");
-        Ok(EngineApi::exchange_transition_configuration(self, config).await?)
+        let start = Instant::now();
+        // TODO: might not be worth including this
+        let res = EngineApi::exchange_transition_configuration(self, config).await;
+        self.metrics.exchange_transition_configuration.record(start.elapsed());
+        Ok(res?)
     }
 
     /// Handler for `engine_exchangeCapabilitiesV1`
     /// See also <https://github.com/ethereum/execution-apis/blob/6452a6b194d7db269bf1dbd087a267251d3cc7f8/src/engine/common.md#capabilities>
     async fn exchange_capabilities(&self, _capabilities: Vec<String>) -> RpcResult<Vec<String>> {
-        Ok(CAPABILITIES.into_iter().map(str::to_owned).collect())
+        let start = Instant::now();
+        // TODO: might not be worth including this
```

Comment:
including this in what?

this is a constant, so probably not particularly useful

Reply:
included as in, should we even have a metric for it at all, will remove

Reply:
gotcha,

I don't think it's very useful